bin/build-ubuntu.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# This script buildd BlazingMQ and all of its dependencies.
+#
+# Install the prerequisites, if not present yet:
+<<PREREQUISITES
+sudo apt update && sudo apt -y install ca-certificates
+sudo apt install -y --no-install-recommends \
+    build-essential \
+    curl \
+    python3.10 \
+    cmake \
+    ninja-build \
+    pkg-config \
+    bison \
+    libfl-dev \
+    libbenchmark-dev \
+    libz-dev
+PREREQUISITES
+
+set -e
+set -u
+set -x
+[ -z $BASH ] || shopt -s expand_aliases
+
+
+# :: Set some initial constants :::::::::::::::::::::::::::::::::::::::::::::::
+DIR_ROOT="${DIR_ROOT:-`pwd`}"
+
+DIR_THIRDPARTY="${DIR_ROOT}/thirdparty"
+mkdir -p ${DIR_THIRDPARTY}
```

Comment:
See my comments on https://github.com/bloomberg/blazingmq/pull/12#pullrequestreview-1550547772; this line and a few others need to expand variables in quotes in case `$DIR_ROOT` is a poorly behaved path with spaces in it.

Reply:
Done everywhere.